styleguide/javascript/README.md

```diff
@@ -82,22 +88,123 @@ Use arrows for functions that are passed as callbacks, since arrow functions bin
 
 If you're using function-style components in React, you (usually) don't need `this`, so it's better to use named functions because they are easier to read. But always prefer arrows instead of `.bind(this)`, and for unnamed closures.
 
+### camelCase in object properties
```

Comment:
I'm not sure I agree with this. I seem to remember such tools making things a bit messier and less obvious than just keeping snake case. But I don't feel very strongly (I only remember it vaguely…) and I'm OK with us trying it out :)

Reply:
Works fine, just make sure you wrap all your API code in it and make it transparent.

With React it's also super-easy to inspect whatever data you're dealing with, so any issues you might have had with other libraries in the past might not even exist in React.

Reply:
camelCase is used for properties almost everywhere in the javascript world. Encountering snake_case all of a sudden just feels really weird...

Also helps with destructuring:

```
const { imageUrls } = item
// is more convenient than
const { image_urls: imageUrls } = item
// and this next one will make eslint angry
const { image_urls } = item

Reply:
Good points. To be clear, I'm 👍  on this part :)